docs/source/apps/apps.rst

```diff
@@ -25,6 +25,17 @@ The recommended way to create a new NMF App project is to copy one of the exampl
 So in our case, we will copy the camera example project folder and rename it to something more fitting, e.g. 'sobel'.
 To make our life easier when we import the project into our workspace, we should edit the app's POM to give it a unique name and artifact ID.
 So in the ``pom.xml`` inside the folder **sobel** change the field ``artifactId`` from the value 'camera' to 'sobel'. You should also change the values of the ``name`` and ``description`` tags to relate to our example and put your name into the ``author`` tags. 
+If you need to use OreKit in your App and don't want to use custom orekit-data, than you sould alsow add the following dependency:
```

Comment:
s/than/then/

Reply:
s/sould/should/

Reply:
s/alsow/also/

Reply:
@dmarszk 
I suggest to copy the content in each XML file and drop it in Word. Then you will immediately see all the typos and mistakes! Should be faster that way!